eq_cir_proxy_service/services/instrument/instrument_retrieval_service.py

```diff
@@ -18,28 +26,44 @@ async def retrieve_instrument(instrument_id: UUID, target_version: str) -> Instr
     Returns:
     - dict: The retrieved instrument.
     """
-    logger.info("Retrieving the instrument...")
-
-    # TODO: Implement the call to CIR to retrieve the instrument
-    # For now, return a dummy instrument
-    instrument = {
-        "language": "en",
-        "mime_type": "application/json/ons/eq",
-        "schema_version": "0.0.1",
-        "data_version": "0.0.1",
-        "survey_id": "999",
-        "form_type": "0001",
-        "title": "Dummy Instrument",
-        "id": instrument_id,
-        "validator_version": "1.0.0",
-        "theme": "business",
-        "metadata": [
-            {"name": "ru_ref", "type": "string"},
-            {"name": "ru_name", "type": "string"},
-        ],
-    }
-
-    logger.debug("Instrument validator version: %s", instrument["validator_version"])
-    logger.debug("Required validator version: %s", target_version)
-
-    return dict(instrument)
+    logger.info("Retrieving instrument %s from CIR...", instrument_id)
+
+    cir_base_url = os.getenv("CIR_API_BASE_URL")
+    cir_endpoint = os.getenv("CIR_RETRIEVE_CI_ENDPOINT")
+    url = f"{cir_base_url}{cir_endpoint}"
+    try:
+        async with httpx.AsyncClient(timeout=10) as client:
+            response = await client.get(url, params={"guid": str(instrument_id)})
+    except httpx.RequestError as e:
```

Comment:
```suggestion
    except RequestError as e:
```
Removes `httpx.` prefix based on previous review comment.